src/signals/IncidentMap/components/IncidentMap/IncidentMap.test.tsx

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2022 Gemeente Amsterdam
+import { render, screen } from '@testing-library/react'
+
+import useFetch from 'hooks/useFetch'
+import configuration from 'shared/services/configuration/configuration'
+import { withAppContext } from 'test/utils'
+
+import { get, mockUseMapInstance, useFetchResponse } from '../__test__/utils'
+import { IncidentMap } from './IncidentMap'
+
+jest.mock('@amsterdam/react-maps', () => ({
+  __esModule: true,
+  ...jest.requireActual('@amsterdam/react-maps'),
+  useMapInstance: jest.fn(() => mockUseMapInstance),
+}))
+
+jest.mock('hooks/useFetch')
+
+describe('IncidentMap', () => {
+  beforeEach(() => {
+    get.mockReset()
+    jest.mocked(useFetch).mockImplementation(() => useFetchResponse)
+  })
+
+  it('should render the incident map correctly', () => {
+    render(withAppContext(<IncidentMap />))
+
+    expect(screen.getByTestId('incidentMap')).toBeInTheDocument()
+  })
+
+  it('sends a request to fetch publicly available incidents', () => {
+    render(withAppContext(<IncidentMap />))
+
+    expect(get).toHaveBeenCalledWith(
+      expect.stringContaining(configuration.GEOGRAPHY_PUBLIC_ENDPOINT)
+    )
+  })
+
+  it('shows a message when the API returns an error', () => {
+    jest.mocked(useFetch).mockImplementation(() => ({
+      ...useFetchResponse,
+      data: undefined,
+      error: new Error(),
+    }))
+
+    render(withAppContext(<IncidentMap />))
+
+    expect(
+      screen.getByText('Er konden geen meldingen worden opgehaald.')
+    ).toBeInTheDocument()
+  })
+})
```

Comment:
consider adding a click interaction on the map message that will close it and add    
```
expect(
       screen.getByText('Er konden geen meldingen worden opgehaald.')
     ).not.toBeInTheDocument()
```
 to make sure that's working as well!